File: A_I_Wanna_Be_the_Guy.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int n;cin>>n;
    int arr[n] = {0};
    int m;cin>>m;
    for(int i=0;i<m;i++){
        int x;cin>>x;
        arr[x-1] = 1;
    }
    int o;cin>>o;
    for(int i=0;i<o;i++){
        int x;cin>>x;
        arr[x-1] = 1;
    }
    for(int i=0;i<n;i++){
        if(arr[i] != 1){
            cout<<"Oh, my keyboard!"<<endl;
            return 0;
        }
    }
    cout<<"I become the guy."<<endl;
return 0;
}

File: A_I_Wanna_Be_the_Guy.java
import java.util.*;
public class A_I_Wanna_Be_the_Guy{
    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        int n = sc.nextInt();
        int a = sc.nextInt();
        int e [] = new int [n];
        for(int i=0;i<n;i++){
            e[i] = i+1;
        }
        int b [] = new int [a];
        for(int i=0;i<a;i++){ 
            b[i] = sc.nextInt();
            for(int j=0;j<n;j++){
                if(b[i]==e[j]){
                    e[j] = 0;
                    break;
                }
            }

        }
        int c = sc.nextInt();
        int d [] = new int [c];
        for(int i=0;i<c;i++){
            d[i] = sc.nextInt();
            for(int j=0;j<n;j++){
                if(d[i]==e[j]){
                    e[j] = 0;
                    break;
                }
            }
        }
        for(int i=0;i<n;i++){
            if(e[i]!=0){
                System.out.print("Oh, my keyboard!");
                return;
            }
        }
        System.out.print("I become the guy.");
    }
}